packages/api/src/schema/api-schema.yaml

```diff
@@ -3799,17 +4093,21 @@ paths:
                   playtimeMins:
                     $ref: >-
                       #/components/schemas/viewership-metric/properties/playtimeMins
+                x-speakeasy-name-override: data
         default:
           description: Error
           content:
             application/json:
               schema:
                 $ref: "#/components/schemas/error"
-  "/data/usage/query":
+  /data/usage/query:
     get:
+      operationId: getUsageMetrics
+      x-speakeasy-name-override: getUsage
+      tags:
+        - metrics
```

Comment:
And here
```suggestion
      operationId: getUsageMetrics
      x-speakeasy-name-override: getMetrics
      tags:
        - usage
```

Reply:
same, i will open a separate PR to update the tags